session_utils/wscript

```diff
@@ -86,7 +86,7 @@ def build(bld):
     utils = bld.path.ant_glob('*.cc', excl=['example.cc', 'common.cc'])
 
     for util in utils:
-        fn = str(util)[:-3]
+        fn = os.path.basename(str(util))[:-3]
```

Comment:
I would suggest to use `os.path.splitext(os.path.basename(str(util)))[0]` here.

(And maybe do `from os.path import basename, dirname, join, normpath, splitext, ...` at the top of the script.)

Reply:
Fair point! I was just being pragmatic ;-)